content/en/cloud_cost_management/multisource_querying/_index.md

```diff
@@ -0,0 +1,131 @@
+<div class="alert alert-warning">Cloud Cost Management is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+<div class="alert alert-warning">Multisource Querying is in private beta. To request access, contact your Customer Success Manager.</div>
+
+## Overview
+
+After you start ingesting your [AWS][1], [Azure][2], [Google Cloud][3], [SaaS][4], or [Datadog costs][5] in [Cloud Cost Management][6], you can flexibly query costs across providers. Multisource Querying enables you to query costs across multiple providers using consistent and standardized tags instead of needing to build multiple queries for each provider.
+
+Use Multisource Querying to build cost views, understand the total cost of service ownership, and alert on cost changes and trends across the [**Analytics** page][6], [dashboards][7], [notebooks][8], and [cost monitors][9].
+
+## Setup
+
+To use Multisource Querying, ensure you meet the following requirements:
+
+1. You have configured [Cloud Cost Management][10] and are actively ingesting costs in Datadog.
+1. Your cost data is in a single currency. Multiple currencies are not supported. 
+
+## Query your cost data
+
+You can select multiple providers in the `Provider` field on the [**Analytics** page][6].
+
+{{< img src="cloud_cost/multisource_querying/provider.png" alt="The Provider field below the search query on the Cloud Cost Analytics page" style="width:40%;" >}}
+
+Dropdown filters like `Provider` maintain flexibility and streamline the process of creating a search query so you can refine your cost data. To add a filter, click **+ Filter**. 
+
+{{< img src="cloud_cost/multisource_querying/filters.png" alt="The Service and Team filters highlighted below the search query on the Cloud Cost Analytics page" style="width:100%;" >}}
+
+Click **Refine Results** to access the following options and filter your cost data.
+
+Usage Charges Only
+: Examine costs impacted by engineering teams, excluding credits, fees, and taxes.
+
+Complete Days Only
+: Exclude the past two days of cost data that are incomplete.
+
+Total Cost
+: Filter the data to view costs within a specific cost range.
+
+Dollar Change
+: Only display cost changes within a specified dollar change range.
+
+Percent Change
+: Only display cost changes within a specified percentage change range.
+
+{{< img src="cloud_cost/multisource_querying/refine_results.png" alt="Additional options to refine your cost data on the Cloud Cost Analytics page" style="width:100%;" >}}
+
+## Visualize your cost data
+
+With Multisource Querying, you can create visualizations using cost data across providers in your [dashboards][11].
+
+{{< img src="cloud_cost/multisource_querying/cost_overview.png" alt="A dashboard in Datadog displaying Cloud Cost Management data from multiple providers such as Snowflake, Azure, Google Cloud, AWS, and more" style="width:100%;" >}}
+
+## Data Collected
```

Comment:
```suggestion
## Data collected
```
I feel like we're inconsistent about this (and Further reading / Further Reading) so I don't know whether you want to fix it or not, just thought I'd flag it